apps/web/src/components/RecruitmentAvailabilityDialog/RecruitmentAvailabilityDialog.tsx

```diff
@@ -0,0 +1,192 @@
+        >
+          {intl.formatMessage({
+            defaultMessage: "Change your availability",
+            id: "33wMLU",
+            description:
+              "Button text to open form to change availability in a generic recruitment",
+          })}
+        </Button>
+      </Dialog.Trigger>
+      <Dialog.Content>
+        <Dialog.Header>
+          {intl.formatMessage({
+            defaultMessage:
+              "Change your availability in this recruitment process",
+            id: "OF7SCg",
+            description:
+              "Header text for dialog to change availability in a recruitment",
+          })}
+        </Dialog.Header>
+        <Dialog.Body>
+          <FormProvider {...methods}>
+            <form onSubmit={handleSubmit(updateSuspendedAtStatus)}>
+              <p data-h2-margin-bottom="base(x1)">
+                {intl.formatMessage(
+                  {
+                    defaultMessage:
+                      "As a successful member of the <strong>{title}</strong> recruitment process, you may be contacted about relevant opportunities.",
+                    id: "mHTt67",
+                    description:
+                      "Text describing what it means to be available in a recruitment",
+                  },
+                  {
+                    title: title.html,
+                  },
+                )}
+              </p>
+              <p data-h2-margin-bottom="base(x1)">
+                {intl.formatMessage({
+                  defaultMessage:
+                    "If you've recently taken a job or simply no longer want to be considered for opportunities, you can disable your availability in this recruitment process. <strong>This will <emphasize>not</emphasize> remove you from the recruitment</strong> itself and you can always re-enable your availability if you change your mind.",
+                  id: "3AwFyW",
+                  description:
+                    "Text describing what it means to disable your availability and that you can reverse the decision",
+                })}
+              </p>
+              <RadioGroup
+                legend={intl.formatMessage({
+                  defaultMessage: "Your availability",
+                  id: "jaMIil",
+                  description:
+                    "Label for available for opportunities radio group",
+                })}
+                idPrefix="availability"
+                id="isSuspended"
+                name="isSuspended"
+                items={[
+                  {
+                    label: intl.formatMessage({
+                      defaultMessage:
+                        "I am <strong>available</strong> for hire and want to be contacted about opportunities from this recruitment process.",
+                      id: "cAOf3a",
+                      description:
+                        "Radio button label for available for opportunities option",
+                    }),
+                    value: "true",
```

Comment:
This works, but it is a little confusing to follow as `isSuspended` is essentially flipped for the form where `true` maps to not suspended.
Connects to this bit below where `isSuspended = true` sets `isSuspended = false`
Not a big deal as it appears to work fine, maybe just a comment or renaming the second variable/form to `isActive` if you want. 

https://github.com/GCTC-NTGC/gc-digital-talent/blob/91711484d7e4e5374c44d8167bab670493973d2b/apps/web/src/components/RecruitmentAvailabilityDialog/RecruitmentAvailabilityDialog.tsx#L32-L33

Reply:
Oof, yeah. That is wrong 😓 I got confused because the form is a positive statement "I am available" so I was thinking positive statement === true 😜 

Fixed in [invert isSuspended logic for availability dialog](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6940/commits/f891ecb079c5be9bab6b770e56d911ff9b22f00e)